pkg/auth/authenticator/password/htpasswd/htpasswd.go

```diff
@@ -116,82 +115,3 @@ func (a *Authenticator) loadIfNeeded() error {
 	}
 	return nil
 }
-
-func testPassword(password, hash string) (bool, error) {
```

Comment:
Yeah, perhaps a better name for the functions would be good, but you can't move them